src/GameLogic/PlayerActions/Chat/IChatProcessorMessage.cs

```diff
@@ -0,0 +1,19 @@
+﻿// <copyright file="IChatProcessorMessage.cs" company="MUnique">
+// Licensed under the MIT License. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace MUnique.OpenMU.GameLogic.PlayerActions.Chat;
+
+/// <summary>
+/// Interface for a chat message processor.
+/// </summary>
+public interface IChatProcessorMessage
```

Comment:
Can you rename these types to "chat message processor", and not "chat processor message" ?